### DISCUSSION TITLE
Options hidden in Vivaldi (Chromium-based) vs Chrome (and other Chromium-based browsers)

### DISCUSSION THREAD

**Discussion by serif** [darkreader/darkreader#7999] [Q&A]
Issue: **I like being able to customize the font color and wish I could do that in Vivaldi.**

In all browsers I have the latest Dark Reader version: 4.9.43

In Chrome (latest 98.0.4758.55) and Edge (latest 97.0.1072.62), I get an interface with a "**See all options**" button, where I can then customize colors:
![Screen Shot 2022-01-19 at 8 05 29 PM](https://user-images.githubusercontent.com/62613995/150246256-99334037-ce12-45f1-a7cb-6c1a8142e874.png)

In Vivaldi (latest snapshot 5.1.2553.3), same 4.9.43 Dark Reader version, I see a different interface which lacks that option:
![Screen Shot 2022-01-19 at 8 04 15 PM](https://user-images.githubusercontent.com/62613995/150246357-10a3f720-4bcd-4e3e-bd47-fed9b8c7ab19.png)

I even copied the extension files from the Chrome version over the one in Vivaldi, and there was no change, so is this intentional? Is there no way around it? 

FYI in both browsers the extension was located under [...]/Default/Extensions/eimadpbcbfnmbkopoojfekhnkhdbieeh/4.9.43_0

The interface is the same regardless of the page I'm on when I open it.

I'm on macOS.

**Comment by Gusted** [comment]
You'd have to enable the new menu:


    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.
